secadvisor/pkg/transform.go

```diff
@@ -238,7 +243,9 @@ func (ft *securityAdvisorFlowTransformer) Transform(f *flow.Flow) interface{} {
 
 	peerContexts := ft.getPeerContexts(f)
 
-	return &SecurityAdvisorFlow{
+	extendMap := make(map[string]interface{})
```

Comment:
no need for variable `extendMap` just set struct field directly

Reply:
OK